e2e-test/helper.py

```diff
@@ -0,0 +1,165 @@
+
+import json
+import uuid
+import random
+import string
+import wds_client
+import requests
+import time
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+workspace_manager_url = ""
+rawls_url = ""
+leo_url = ""
+
+def setup(bee_name):
+# define major service endpoints based on bee name
+    global workspace_manager_url
+    workspace_manager_url = f"https://workspace.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"workspace_manager_url: {workspace_manager_url}")
+    global rawls_url
+    rawls_url = f"https://rawls.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"rawls_url: {rawls_url}")
+    global leo_url
+    leo_url = f"https://leonardo.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"leo_url: {leo_url}")
+    return workspace_manager_url, rawls_url, leo_url
+
+# CREATE WORKSPACE ACTION
+def create_workspace(cbas, billing_project_name, header):
+    # create a new workspace, need to have attributes or api call doesnt work
+    api_call2 = f"{rawls_url}/api/workspaces";

```

Comment:
sorry this is totally my fault, but this can probably have a better name :D

Reply:
there are a few other ambiguous names in this file that can be improved, I think for clarify it would be useful to rename all the smthNumber to something more descriptive